setup.cfg

```diff
@@ -0,0 +1,13 @@
+[pydocstyle]
+convention = google
```

Comment:
I suggest using autodoc convention since we are using `Sphinx`

Reply:
Done